### PR TITLE
Lcam 911

### PR DESCRIPTION
**Pull request by venkat980**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-911)

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by skasthuri36326 — APPROVED** [review]

File: maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java [hunk 1]
```diff
@@ -39,7 +43,10 @@ public ResponseEntity<HardshipReviewDTO> find(
             @Valid @RequestBody GetHardshipDTO getHardshipDTO,
             @Parameter(description = "Used for tracing calls") @RequestHeader(value = LAA_TRANSACTION_ID, required = false) String laaTransactionId) {
         log.info("Received request to find hardship with transaction id - " + laaTransactionId);
-        return ResponseEntity.ok(HardshipReviewDTO.builder().build());
+        Integer hardshipAssessmentId = getHardshipDTO.getHardshipAssessmentId();
+        HardshipReviewDTO hardshipReviewDTO = HardshipReviewDTO.builder().id(hardshipAssessmentId.longValue()).build();
+        hardshipReviewMapper.toDto(hardshipService.getHardship(hardshipAssessmentId), hardshipReviewDTO);
```
[the remark this answers is not in the document]
Comment (venkat980) [reply maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java:48]: It is an overridden method.

File: maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java [hunk 2]
```diff
@@ -39,7 +43,10 @@ public ResponseEntity<HardshipReviewDTO> find(
             @Valid @RequestBody GetHardshipDTO getHardshipDTO,
             @Parameter(description = "Used for tracing calls") @RequestHeader(value = LAA_TRANSACTION_ID, required = false) String laaTransactionId) {
         log.info("Received request to find hardship with transaction id - " + laaTransactionId);
-        return ResponseEntity.ok(HardshipReviewDTO.builder().build());
+        Integer hardshipAssessmentId = getHardshipDTO.getHardshipAssessmentId();
```
[the remark this answers is not in the document]
Comment (venkat980) [reply maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java:46]: Just to be consistent, we are keeping all the ids as Integers.

File: maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java [hunk 3]
```diff
@@ -39,7 +43,10 @@ public ResponseEntity<HardshipReviewDTO> find(
             @Valid @RequestBody GetHardshipDTO getHardshipDTO,
```
[the remark this answers is not in the document]
Comment (venkat980) [reply maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/controller/HardshipOrchestrationController.java:43]: Thought of naming it as GetHardshipDTORequest, but kept it in line with other DTO classess.

**Comment by muthusubra** [comment]
Also, there are 3 code smells. Please resolve

**Pull request merged**